samples/csharp_dotnetcore/12.customQABot/CustomQABot.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <LangVersion>latest</LangVersion>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="3.1.1" />
+    <PackageReference Include="Microsoft.Bot.Builder.AI.QnA" Version="4.16.0-daily.20220310.295385.f7b481e" />
```

Comment:
@gabog should the version be fixed to 4.16.0. Or this file will be reworked before merge to main?

Reply:
We will take care of updating this to 4.16 when we merge into main.